external/grpc/Kconfig.protocol

```diff
@@ -13,6 +13,12 @@ config GRPC
 
 if GRPC
 
+config GRPC_PTHREAD_SIZE
```

Comment:
Is this gRPC main thread? or sub-threads made by gRPC main thread?

Reply:
sub-threads created for grpc core modules.

Reply:
As you know, usually developer does not see documentation. How about mark this with "sub-threads" or something else which we can know what this configures?
And how about add description how to set gRPC main thread stack size in ---help--- tab.